sentinel/src/transitions/accept_patient_reports.js

```diff
@@ -103,7 +103,14 @@ const _silenceReminders = (registration, report, config, callback) => {
     utils.setTaskState(task, 'cleared');
     task.cleared_by = report._id;
   });
-  db.medic.post(registration, callback);
+  return db.medic.post(registration, function(err, response) {
+    if (err) { 
+      return callback(err);
+    }
+    
+    registration._rev = response._rev;
```

Comment:
The `response` object will not have a `_rev` property, instead a `rev` property, and you'll still have a conflict.
Please see https://pouchdb.com/api.html#create_document or http://docs.couchdb.org/en/stable/api/document/common.html#put--db-docid for example responses.